.github/workflows/build-test-package.yml

```diff
@@ -66,7 +66,7 @@ jobs:
     steps:
     - uses: actions/checkout@master
     - name: Set up Python "3.9"
-      uses: actions/setup-python@v1
+      uses: actions/setup-python@v2
```

Comment:
Is there a reason not to jump all the way to the latest version?  v5 exists, for instance.

In other projects, we use dependabot to update actions. 

And, I'll note that the pypi-publish action recommends a version of release/v1 rather than master.

Reply:
I was aiming for the minimal change that works, to minimize the chance of breaking something else.  Though, if `v5` is expected to backward compatible ... I'll make that change.

Reply:
> In other projects, we use dependabot to update actions.

I don't know anything about how to do that.  If that is something you can do in your sleep, that'd be great!  Otherwise, I'd appreciate your opinion on the relative importance of pursuing that.

Reply:
You add a dependabot config file in the .githib folder (see https://github.com/DigitalSlideArchive/superpixel-classification/blob/main/.github/dependabot.yml as an example -- you can just copy that).  Then on some schedule if there are updates to the actions, dependabot will make PRs.

Reply:
I have added `.github/dependabot.yml` as a copy of the one from `DigitalSlideArchive/superpixel-classification`.

Reply:
> And, I'll note that the pypi-publish action recommends a version of release/v1 rather than master.

Thank you.  Done.